forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -54,6 +54,14 @@ pub enum SnapshotCommands {
         /// in default Forest data location.
         #[structopt(short, long)]
         snapshot_dir: Option<PathBuf>,
+        /// Fetch latest snapshot provided by `forest` | `filecoin`
+        #[structopt(
+            short,
+            long,
+            default_value = "forest",
+            possible_values = &["forest", "filecoin"],
+        )]
+        provider: String,
```

Comment:
How about making this flag a `SnapshotServer` instead of a `String`? Or possibly an `Option<SnapshotServer>` which defaults to `Forest` for calibnet and `Filecoin` for mainnet.

Reply:
Fixed